packages/bitcore-node/src/modules/ethereum/api/transform.ts

```diff
@@ -14,12 +14,13 @@ export class EthListTransactionsStream extends Transform {
       wallet: this.wallet._id,
       address: transaction.from
     });
-    if (sending > 0) {
+
+    if (sending > 0 || transaction.from === this.multisigContractAddress) {
       const sendingToOurself = await WalletAddressStorage.collection.countDocuments({
         wallet: this.wallet._id,
         address: transaction.to
       });
-      if (!sendingToOurself) {
+      if (!sendingToOurself || transaction.to !== this.multisigContractAddress) {
```

Comment:
If the address is the multisigContractAddress, this wouldn't need to be added.

Reply:
Same here... there is no wallet data stored for multisig. So, 
```
const sendingToOurself = await WalletAddressStorage.collection.countDocuments({
        wallet: this.wallet._id,
        address: transaction.to
      });
```
will not find any coincidence.